packages/@aws-cdk/aws-eks/lib/kubectl-handler/helm/__init__.py

```diff
@@ -109,8 +109,8 @@ def get_oci_cmd(repository, version):
             f"helm registry login --username AWS --password-stdin {registry}; helm pull {repository} --version {version} --untar"
             ]
     elif registry.startswith(public_ecr):
-        logger.info("Found AWS public repository, will use default region as deployment")
-        region = os.environ.get('AWS_REGION', 'us-east-1')
+        logger.info("Found AWS public repository, will use region 'us-east-1' as deployment")
+        region = 'us-east-1'
```

Comment:
There's two valid regions, `us-east-1` and `us-west-2`. If someone today is using `us-west-2`, then this change will force them to redeploy to update to `us-east-1`. We try to avoid forcing new deployments in new releases, so can you update this to check if `AWS_REGION` is `us-west-2`, and if it is, continue to use `us-west-2`?

Reply:
@comcalvi 

According to the Amazon ECR public registries documentation the region should always be `us-east-1` for the ecr-public get-login-password operation.

https://docs.aws.amazon.com/AmazonECR/latest/public/public-registries.html#public-registry-auth
When authenticating to a public registry, always authenticate to the us-east-1 Region when using the AWS CLI.

so I think we should always make it `us-east-1` here, even deploying in `us-east-2`.
https://github.com/aws/aws-cdk/blob/c0b10195b900fff6f481f4d94aa182d8e7278d00/packages/%40aws-cdk/aws-eks/lib/kubectl-handler/helm/__init__.py#L111-L121